desktop/hardened-desktop/admin-controls/configure-ac.md

```diff
@@ -0,0 +1,144 @@
+---
+description: admin controls for desktop
+keywords: admin, controls, rootless, enhanced container isolation
+title: Configure Admin Controls
+--- 
+
+>Note
+>
+>Admin Controls is available to Docker Business customers only. 
+
+This page contains information about how administrators can configure Admin Controls to specify and lock configuration parameters to create a standardized Docker Desktop environment across the organization.
+
+Admin Controls is designed specifically for organizations who don’t give developers root access to their machines.
+
+## Prerequisite
+
+You need to [configure a registry.json to enforce sign-in](../../../docker-hub/configure-sign-in.md). For this configuration to take effect, Docker Desktop users must authenticate to your organization. 
+
+## Step one: Place the `admin-settings.json` file in the correct location
+
+Place the `admin-settings.json` file on your developers' machines in the following locations:
+
+- Mac: `/Library/Application Support/com.docker.docker/admin-settings.json`
+- Windows: `/ProgramData/DockerDesktop/admin-settings.json`
+- Linux: `/usr/share/docker-desktop/admin-settings.json`
+
+By placing this file in the above protected directories, end users are unable to modify it.
+
+>Note
+>
+> It is assumed that you have the ability to push the `admin-settings.json` settings file to the locations specified above through a device management software such as [Jamf](https://www.jamf.com/lp/en-gb/apple-mobile-device-management-mdm-jamf-shared/?attr=google_ads-brand-search-shared&gclid=CjwKCAjw1ICZBhAzEiwAFfvFhEXjayUAi8FHHv1JJitFPb47C_q_RCySTmF86twF1qJc_6GST-YDmhoCuJsQAvD_BwE).
+
+## Step two: Configure the admin controls you want to lock in
+
+>Note
+>
+>Some of the configuration parameters only apply to Windows. This is highlighted in the table below.
+
+The `admin-settings.json` file requires a nested list of configuration parameters, each of which must contain the  `locked` parameter. You can add or remove configuration parameters as per your requirements.
+If set to `true`, users are not able to edit this setting from Docker Desktop or the CLI. 
+If set to `false`, the configuration value acts as a default value, but users can change this setting from Docker Desktop or the CLI by directly editing the `settings.json` file.
```

Comment:
OK so this + table below are great but I'd restructure slightly:

Paragraph 1 - "The `admin-settings.json` file requires...as per your requirements"

Paragraph 2 - "If `locked:true`, users will not be able to modify this setting via Docker Desktop.

Paragraph 3:

"Setting a value as `locked: false` is similar to setting a factory default, in that:

* For new installs, `locked: false pre-populated the relevant setting in the Docker Desktop UI, but users will be able to modify it as they please

* When Docker Desktop is already installed and being used, `locked: false is ignored. This is because existing users of Docker Desktop may have already updated a setting, which in turn will have been written to the relevant config file, for example the `settings.json` or `daemon.json`. In these instances, the user's preferences are respected and we do not alter these values. These can be controlled by the administrator by setting `locked:true`

Reply:
And then I'd remove the 'Addition locked: false logic' table entirely. Let's just pull out the relevant details from that (which are important) and make them visible.